package.json

```diff
@@ -4,7 +4,7 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
+        "test": "npx hardhat run --network localhost",
```

Comment:
OK I ran this again... localhost won't work unless you run a node... I just use testnet usually. 

I think command should be: `npx hardhat run --network polygonMumbai` or ropsten.

Reply:
Ah... ye, need ran hardhat node. Is it commonto use testnet for testing contract?

Reply:
> 

yeah especially Polygon Mumbai Testnet... Gas is basically $0.00 free, since L2 with MATIC. 
we can use Polygon Mumbai for all testing... easier then localhost.
can get some testnet MATIC here: https://matic.supply/
then update the .env file MNEMONIC with your wallet private phrase which holds the Mumbai MATIC